src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -858,7 +857,7 @@ protected override void Dispose(bool disposing)
             await props.ActiveDebugProfile.SetValueAsync(profileName).ConfigureAwait(true);
         }
 
-        internal async Task<string> GetLaunchSettingsFilePathAsync()
+        internal async Task<string> GetLaunchSettingsFilePathNoCacheAsync()
         {
             // NOTE: To reduce behavior changes, we currently cache the folder that we get from
```

Comment:
>we currently cache the folder [](start = 49, length = 29)

The comment says that we cache the folder but the method name says we do not cache.  Which is it?